### DISCUSSION TITLE
Is there any way to  import GraphQL collection from Postman correctly?

### DISCUSSION THREAD

**Discussion by mikbagautdinov** [hoppscotch/hoppscotch#3268] [Q&A]
Hello! I have a GraphQL collection imported from Postman. Is there any way to add it to hoppscotch? Even though I managed to do it in REST section, it doesn't work as it's GraphQL collection.( I would appreciate any answer.

**Comment by jamesgeorge007** [comment]
Hi, Hoppscotch now supports importing GraphQL requests (natively to the REST section) based on the [Postman HTTP request interface](https://learning.postman.com/docs/sending-requests/graphql/graphql-http/#use-postmans-http-request-for-graphql) via https://github.com/hoppscotch/hoppscotch/pull/4967.
